app/talk/search.cjsx

```diff
@@ -26,6 +26,14 @@ module.exports = React.createClass
   displayName: 'TalkSearch'
   mixins: [History]
 
+  contextTypes:
+    geordi: React.PropTypes.object
+
+  logBackClick: (linkName) ->
+    @history.goBack()
```

Comment:
Method should be renamed - it does more than just log, it actually executes the Back